opentracing/span.py

```diff
@@ -208,17 +210,28 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         """Ends context manager and calls finish() on the :class:`Span`.
 
-        If exception has occurred during execution, it is automatically added
-        as a tag to the :class:`Span`.
+        If exception has occurred during execution, it is automatically logged
+        and added as a tag. :attr:`~operation.ext.tags.ERROR` will also be set
+        to `True`.
         """
-        if exc_type:
-            self.log_kv({
-                'python.exception.type': exc_type,
-                'python.exception.val': exc_val,
-                'python.exception.tb': exc_tb,
-                })
+        self._on_error(exc_type, exc_val, exc_tb)
         self.finish()
 
+    def _on_error(self, exc_type, exc_val, exc_tb):
+        if not exc_type:
+            return
+
+        self.log_kv({
+            'python.exception.type': exc_type,
+            'python.exception.val': exc_val,
+            'python.exception.tb': exc_tb,
+            })
+        self.set_tag(tags.ERROR, True)
+        self.log_kv({
+            'event': tags.ERROR,
+            'error.object': exc_type,
```

Comment:
this duplicates the previous log_kv, would it not make sense to combine?

Reply:
Personally I don't much preference - so we can definitely put all that info in a single place ;)

Reply:
I believe OT spec has the field name for the stacktrace, which takes care of `python.exception.tb`. I don't know what `python.exception.val` is about.

My preference is to converge to OT standard log fields instead of these custom python strings (which predate official OT fields). But it will be a breaking change, in some way. Worth checking with other tracing vendors if they have explicit dependency on these strings in the backend (Jaeger doesn't).

Reply:
Oh, definitely. I had wondered about it too (that being said, it looks logging the error through `error.object` should be enough for us).

Will double check and ask away ;)